gigareview/common/models/EMReportJob.php

```diff
@@ -14,9 +14,82 @@ class EMReportJob extends \yii\base\BaseObject implements \yii\queue\JobInterfac
     public string $fetchDate ;
     public string $scope;
 
+
     public function execute($queue)
     {
-        // TODO: Implement execute() method.
+        if ($this->scope === "manuscripts") {
+            $this->executeManuscriptJob($this->content);
+        }
+            //TODO: Will be implemented in ticket no. #1065
+//            $ingest = Ingest::findOne(["report_type" => "1", "parse_status" => null]);
+//            $ingest->remote_file_status = Ingest::REMOTE_FILES_STATUS_EXISTS;
+//            $ingest->parse_status = Ingest::PARSE_STATUS_YES;
+//            $ingest->update();
+
+//        }
+            //TODO: Will be implemented in ticket no. #1065
+//        elseif ($this->scope === "manuscripts" && $this->content === "No Results") {
+//            $ingest = Ingest::findOne(["report_type" => "1", "parse_status" => null]);
+//            $ingest->remote_file_status = Ingest::REMOTE_FILES_STATUS_NO_RESULTS;
+//            $ingest->parse_status = Ingest::PARSE_STATUS_NO;
+//            $ingest->update();
+//        }
+    }
+
+    /**
+     * @param string $content
+     * @return void
+     */
+    public function executeManuscriptJob(string $content): void
+    {
+        if ($content !== "No Results") {
+            //Step 1: Put queue content to csv
+            $tempManuscriptCsvFile = tempnam(sys_get_temp_dir(), "test-manuscripts").".csv";
```

Comment:
Do you need to call `unlink()` to delete temporary files created by `tempnam()`?

Reply:
The csv file is explicitly stored in the `/tmp` dir, all the files in there will be deleted wherever the system is booted, in our case, will be whenever the `% docker-compose up -d manuscripts-worker` is finished.